### ISSUE TITLE
Implement Multi-page Modals

### ISSUE THREAD

**Issue by smalyon-scottlogic** [ScottLogic/prompt-injection#578]
Add multi-page modal for Get Started (more?), e.g. 
New design with page indication and previous and next navigation buttons: 
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/d4eb52a3-46db-4e6e-b922-85eb1c072797)

any action buttons should be above the navigation buttons:
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/e4ad15c1-d4cf-4eab-8ea7-1d4eef3e7853)

**Comment by gsproston-scottlogic** [comment]
Don't really need new UI designs, just split up what we have into multi-page modals. 
Need wireframes.

**Comment by dhinrichs-scottlogic** [comment]
Wireframes (as a rough suggestion, please use your best judgment for spacing, which icons to use for personas, which side to put them on etc...):
https://docs.google.com/presentation/d/1OvYoXIdhCJCH2PvVxzzuiT43UnTZ5bMnHhBYSpqyfRQ/edit#slide=id.p
[Multi Page Modals.pdf](https://github.com/ScottLogic/prompt-injection/files/13832565/Multi.Page.Modals.pdf)

Also, I forgot to add the "OK" Button in, but that's already part of our modals, so please keep them in. (maybe only the 4th modal but keep the X close button on each modal?)

**Comment by chriswilty** [comment]
@m4rkwallace-SL Do you have time to give any feedback on the wireframes / provide some design guidance? If not, we'll put something together and then show you when you have time.
